### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by genusistimelord**
unfinished work

### CODE REVIEW

File: src/native/date_picker.rs [hunk 1]
```diff
@@ -53,7 +53,7 @@ pub struct DatePicker<'a, Message: Clone, Renderer: iced_native::Renderer> {
     /// The function that produces a message when the submit button of the [`DatePickerOverlay`](DatePickerOverlay) is pressed.
     on_submit: Box<dyn Fn(Date) -> Message>,
     /// The style of the [`DatePickerOverlay`](DatePickerOverlay).
-    style_sheet: Box<dyn StyleSheet + 'a>,
+    style_sheet: Box<dyn StyleSheet + 'static>,
```
Comment (Kaiden42) [inline src/native/date_picker.rs]: why does the style sheet needs to be static here?

Comment (genusistimelord) [reply src/native/date_picker.rs]: idk it just would not compiler till i made it static. Ill try it without

File: src/native/overlay/date_picker.rs [hunk 2]
```diff
@@ -54,7 +53,7 @@ where
     /// The position of the [`DatePickerOverlay`](DatePickerOverlay).
     position: Point,
     /// The style of teh [`DatePickerOverlay`](DatePickerOverlay).
-    style_sheet: &'a Box<dyn StyleSheet + 'a>,
+    style_sheet: &'a Box<dyn StyleSheet + 'static>,
```
Comment (Kaiden42) [inline src/native/overlay/date_picker.rs]: why does the style sheet needs to be static here too?

**Review by Kaiden42 — CHANGES_REQUESTED** [review]
Great! Only two little comments.

**Review by Kaiden42 — APPROVED** [review]
Thanks!

**Pull request merged**